pkg/cfn/builder/managed_launch_template.go

```diff
@@ -50,7 +51,23 @@ func (m *ManagedNodeGroupResourceSet) makeLaunchTemplateData() (*gfnec2.LaunchTe
 		}
 	}
 
-	launchTemplateData.SecurityGroupIds = gfnt.NewValue(securityGroupIDs)
+	if api.IsEnabled(mng.EFAEnabled) {
+		// we don't want to touch the network interfaces at all if we have a
+		// managed nodegroup, unless EFA is enabled
+		if err := buildNetworkInterfaces(launchTemplateData, mng.BaseNodeGroup(), securityGroupIDs, m.ec2API); err != nil {
+			return nil, errors.Wrap(err, "couldn't build network interfaces for launch template data")
+		}
+		if mng.Placement == nil {
+			groupName := m.newResource("NodeGroupPlacementGroup", &gfnec2.PlacementGroup{
+				Strategy: gfnt.NewString("cluster"),
+			})
+			launchTemplateData.Placement = &gfnec2.LaunchTemplate_Placement{
+				GroupName: groupName,
+			}
```

Comment:
do we want to make configuraing placement and enabling EFA an invalid ? Or document & warn that if they are enabling EFA and providing there own placement group they need to ensure the placement group has x/y for it to work with EFA?

Reply:
I don't think it's worth it, since we'd need to fetch the group and figure out its type and validate other requirements. If a user sets EFA they should know how it interacts with placement groups, this is just a shortcut to avoid having to _create_ a placement group

Reply:
`they should know how nodegroups (including EFA) interacts with placement groups` BIG assumption :sweat_smile: